src/commands/runDraftTool/runDraftDockerfile.ts

```diff
@@ -109,8 +109,14 @@ class PromptSourceCodeFolder extends AzureWizardPromptStep<WizardContext> {
 class PromptLanguage extends AzureWizardPromptStep<WizardContext> {
    public async prompt(wizardContext: WizardContext): Promise<void> {
       const languageToItem = (lang: DraftLanguage) => ({label: lang.name});
+      let infoLanguages: DraftLanguage[];
+      try {
+         infoLanguages = await getDraftLanguages();
+      }catch (error) {
+         throw Error('Failed to get Draft languages');
```

Comment:
it would be better to wrap the error message inside `getDraftLanguages()`. That way you don't need to catch it everywhere you use it to properly handle and add context to the error.